frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -130,6 +161,28 @@ export const BuySell = ({
                 onClickInfoButton={() => setInfo(buildInfo(exchange))}
               />
             ))}
+            {exchangeDealsResponse?.exchanges.some(exchange => exchange.exchangeName === 'pocket') && (
```

Comment:
the problem with this is that Pocket doesnt sell altcoins and so therefore BTCDirect never shows up if an altcoin account was selected.